analyses/TX_cd_2020/03_sim_TX_cd_2020.R

```diff
@@ -0,0 +1,445 @@
+###############################################################################
+# Simulate plans for `TX_cd_2020`
+# © ALARM Project, February 2022
+###############################################################################
+set.seed(02138)
+library(sf)
+library(tidyverse)
+library(patchwork)
+
+source("analyses/TX_cd_2020/TX_helpers.R")
+source("analyses/TX_cd_2020/01_prep_TX_cd_2020.R")
+source("analyses/TX_cd_2020/02_setup_TX_cd_2020.R")
+
+cluster_pop_tol <- 0.0025
+
+map <- set_pop_tol(map, 0.01)
```

Comment:
change l. 7 of `02_setup` to have `pop_tol = 0.01` rather than setting here

Reply:
Is 0.01 the best we can do here?

Reply:
I'm not wedded to needing 0.5% in every state, but if you're able/willing and Chris thinks it's worth it, then we might as well...?